Formula/clozure-cl.rb

```diff
@@ -0,0 +1,35 @@
+class ClozureCl < Formula
+  desc "Common Lisp implementation with a long history"
+  homepage "https://ccl.clozure.com"
+  url "https://github.com/Clozure/ccl/releases/download/v1.11.5/ccl-1.11.5-darwinx86.tar.gz"
+  sha256 "5adbea3d8b4a2e29af30d141f781c6613844f468c0ccfa11bae908c3e9641939"
+  head "https://github.com/Clozure/ccl.git"
+
+  depends_on :xcode => :build
+
+  def install
+    cd "lisp-kernel/darwinx8664" do
+      system "make", "clean"
+      system "make"
+    end
+
+    ENV["CCL_DEFAULT_DIRECTORY"] = buildpath
+    system "scripts/ccl64", "-n",
+           "-e", "(ccl:rebuild-ccl :full t)",
+           "-e", "(quit)"
+
+    prefix.install "doc/README"
+    doc.install Dir["doc/*"]
+
+    libexec.install Dir["*"]
```

Comment:
This is still installing prebuilt binaries.
```
bash-3.2$ find . -type f -exec file "{}" \;|grep Mach
./dx86cl: Mach-O executable i386
./dx86cl64: Mach-O 64-bit executable x86_64
bash-3.2$ 
```

Reply:
@ilovezfs I'm confused... that only happens in the installed directory, `/usr/local/Cellar/clozure-cl/86/`. It does not happen in the source directory, even after a fresh clone via `git clone https://github.com/Clozure/ccl.git`.

Reply:
That command is in the buildpath. You're using the release tarball which has prebuilt binaries unlike https://github.com/Clozure/ccl/archive/v1.11.5.tar.gz

Reply:
@ilovezfs It seems that there is [an expectation](https://github.com/Clozure/ccl/issues/84) from the developers that one has a preexisting heap image in order to compile properly. Even their documentation points to this fact, the docs which you said are "poor." So I need to consult with their developers a bit and see if a fully bootstrapped build is possible.

This is an unfortunate situation for package managers which like to build everything from source, but this *is* the way programs are traditionally built in the Lisp world. The practice dates back to Lisp's inception way back in the 50's. So it's not just some crazy half-baked program which doesn't know how to build itself properly.

In the meantime, please do not close this PR. This is a useful program and it actually is used by a number of Mac users so it would be really nice to have this in homebrew again.

Reply:
You've misunderstood the requirements. Using a bootstrap binary during the build is acceptable. See gcc, crystal-lang, etc. You must not install precompiled binaries, however, which this formula still does.

Reply:
ghc not gcc excuse me

Reply:
@ilovezfs Oh, okay. Thanks for clearing that up. I will try to adapt the formula to do such a thing. Thanks also for pointing me to some examples.

Reply:
More examples: fpc, go, sbcl.

Reply:
@ilovezfs I've got the formula mostly working thanks to your advice (and thanks to [some better documentation](https://ccl.clozure.com/manual/chapter3.html) that I found in the CCL manual). But I've hit a snag I can't seem to get past: how do I send data to a process's standard input in the `install` method? There is a command run in the `install` method which will not accept batch commands from the command line itself; the commands to run must be delivered directly to the process's REPL via the standard input. But I cannot find a way to do this short of using a `PTY`, which it appears that no other formula uses except for testing. Do you know of a way around this?

Reply:
Does `system "echo foo | bar"` work?